ui/src/components/admin/stats/BasicAuthPrompt.vue

```diff
@@ -62,6 +63,28 @@
                             trigger: ["blur"],
                             pattern: "^$|^[a-zA-Z0-9_!#$%&’*+/=?`{|}~^.-]+@[a-zA-Z0-9.-]+$"
                         },
+                        {
+                            validator: (rule, value, callback) => {
+                                if (value && value.length > 256) {
+                                    callback(new Error(this.$t("Email must not exceed 256 characters")));
```

Comment:
For the text of error message, you need to add a new attributes to the end of both https://github.com/kestra-io/kestra/blob/develop/ui/src/translations/en.json and https://github.com/kestra-io/kestra/blob/develop/ui/src/translations/fr.json files (for French, you can use Google translate or any other online service) and then add the key to that translation into the `this.$t("")`, instead of the whole text.